modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaGetFDSTableSchemaParameterCheck.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+import com.liferay.source.formatter.parser.JavaClass;
+import com.liferay.source.formatter.parser.JavaClassParser;
+import com.liferay.source.formatter.parser.JavaMethod;
+import com.liferay.source.formatter.parser.JavaTerm;
+import com.liferay.source.formatter.parser.ParseException;
+
+import java.io.IOException;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Albert Gomes Cabral
+ */
+public class UpgradeJavaGetFDSTableSchemaParameterCheck
+	extends BaseUpgradeCheck {
+
+	@Override
+	protected String format(
+			String fileName, String absolutePath, String content)
+		throws IOException, ParseException {
+
+		String newContent = content;
+
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
+
+		List<String> extendedClassNames = javaClass.getExtendedClassNames();
+
+		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
+			if (!childJavaTerm.isJavaMethod()) {
+				continue;
+			}
+
+			if (extendedClassNames.contains("BaseTableFDSView")) {
+				JavaMethod javaMethod = (JavaMethod)childJavaTerm;
+
+				String javaMethodContent = javaMethod.getContent();
+
+				Matcher matcher = _getFDSTableSchemaPattern.matcher(
+					javaMethodContent);
+
+				while (matcher.find()) {
+					newContent = _addParameter(content, newContent, matcher);
+				}
+			}
+		}
+
+		return newContent;
+	}
+
+	@Override
+	protected String[] getNewImports() {
+		return new String[] {"java.util.Locale"};
+	}
+
+	private String _addParameter(
+		String content, String newContent, Matcher matcher) {
+
+		String methodCall = matcher.group();
+
+		List<String> parameterList = JavaSourceUtil.getParameterList(
+			methodCall);
+
+		if (parameterList.size() == 1) {
+			return newContent;
+		}
```

Comment:
@alberttgomes this is really necessary? We usually use this validation when the regex stops on the first parameter or in other cases. In your case, the regex has explicitly the () at the end, so never it will have a parameter on this match. What you think?